pkgs/by-name/bi/bitbox-bridge/package.nix

```diff
@@ -7,7 +7,7 @@
   libudev-zero,
 }:
 
-rustPlatform.buildRustPackage (finalAttrs: {
+rustPlatform.buildRustPackage (finalAttrs: rec {
```

Comment:
There is no need for `rec` while using `finalAttrs`

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

Reply:
Nope, the rec is required for the src reference below.

Reply:
`finalAttrs.src` does the same, and it is also not needed, you can operate on the `$src` directly without needing to reference its derivation